src/dhcp6relay/src/relay.cpp

```diff
@@ -20,6 +20,66 @@ struct event_base *base;
 struct event *ev_sigint;
 struct event *ev_sigterm;
 
+/* DHCPv6 filter */
+/* sudo tcpdump -dd "ip6 dst ff02::1:2 && udp dst port 547" */
+
+static struct sock_filter ether_relay_filter[] = {
+	
+	{ 0x28, 0, 0, 0x0000000c },
+	{ 0x15, 0, 13, 0x000086dd },
+	{ 0x20, 0, 0, 0x00000026 },
+	{ 0x15, 0, 11, 0xff020000 },
+	{ 0x20, 0, 0, 0x0000002a },
+	{ 0x15, 0, 9, 0x00000000 },
+	{ 0x20, 0, 0, 0x0000002e },
+	{ 0x15, 0, 7, 0x00000000 },
+	{ 0x20, 0, 0, 0x00000032 },
+	{ 0x15, 0, 5, 0x00010002 },
+	{ 0x30, 0, 0, 0x00000014 },
+	{ 0x15, 0, 3, 0x00000011 },
+	{ 0x28, 0, 0, 0x00000038 },
+	{ 0x15, 0, 1, 0x00000223 },
+	{ 0x6, 0, 0, 0x00040000 },
+	{ 0x6, 0, 0, 0x00000000 },
+};
+const struct sock_fprog ether_relay_fprog = {
+	lengthof(ether_relay_filter),
+	ether_relay_filter
+};
+
+/* DHCPv6 Counter */
```

Comment:
can we split counters to be per vlan?

Reply:
Yes. Updated: Each vlan will have its own counter table (ex: DHCPv6_Counter|Vlan1000, DHCPv6_Counter|Vlan2000)